index.js

```diff
@@ -31,7 +31,14 @@
   var _crypto;
   if(typeof crypto !== 'undefined') {
     _crypto = crypto;
+  } else if (window.crypto) {
+    _crypto = window.crypto;
+  } else if (window.msCrypto) {
+    _crypto = window.msCrypto;
```

Comment:
These checks need to be moved to AFTER the Node support check.
I think a single block (maybe in the form of a helper function) would suffice. 
Pseudocode: 
1. Check for node support
2. Check for window.crypto
3. Check for window.msCrypto
One thing to look out for is making sure that we return the control flow as fast as possible.